README.md

```diff
@@ -33,6 +33,36 @@ it with:
 OpenComponents.configure { |config| config.registry = 'http://some.other.host' }
 ```
 
+## Getting Rendered Components
+To request an OpenComponent with its rendered HTML, you can create and load a new
```

Comment:
I actually prefer using passive voice in documentation. Unless it's not consistent, of course.